community/tlp/build

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh -e
+
+export DESTDIR="$1"
+export TLP_WITH_ELOGIND="0"
+export TLP_WITH_SYSTEMD="0"
+export TLP_NO_BASHCOMP="1"
+export TLP_NO_INIT="1"
+export TLP_NO_TPACPI="1"
+
+make
+
```

Comment:
Drop this newline.

Reply:
I'm not sure which one you meant here, I assume you meant the one between make and make install, so I removed that one. If that was not the case then which one?

Reply:
after looking at the resources at k1ss.org it seems this was the case.